json/schemas/whitelabel-schema.json

```diff
@@ -0,0 +1,119 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "White Labelling",
+    "description": "White Labelling Configuration, see section 5.7 in the RP2350 datasheet for more details",
+    "type": "object",
+    "properties": {
+        "$schema": {},
+        "device": {
+            "description": "Device Properties",
+            "type": "object",
+            "properties": {
+                "vid": {
+                    "description": "Vendor ID",
+                    "type": "string",
+                    "pattern": "^0x[0-9a-fA-F]{4}$"
+                },
+                "pid": {
+                    "description": "Product ID",
+                    "type": "string",
+                    "pattern": "^0x[0-9a-fA-F]{4}$"
+                },
+                "bcd": {
+                    "description": "Device Revision",
+                    "type": "number",
+                    "minimum": 0,
+                    "maximum": 99
+                },
+                "lang_id": {
+                    "description": "Language ID",
+                    "type": "string",
+                    "pattern": "^0x[0-9a-fA-F]{4}$"
+                },
+                "manufacturer": {
+                    "description": "Manufacturer Name (can contain unicode)",
+                    "type": "string",
+                    "maxLength": 30
+                },
+                "product": {
+                    "description": "Product Name (can contain unicode)",
+                    "type": "string",
+                    "maxLength": 30
+                },
+                "serial_number": {
+                    "description": "Serial Number (can contain unicode)",
+                    "type": "string",
+                    "maxLength": 30
+                },
+                "max_power": {
+                    "description": "Max power consumption. in 2mA units",
```

Comment:
nitpick: I guess either the `.` should be a `,`, or the `In` should be capitalised?

Reply:
Typo - fixed